tests/Unit/ErrorHandling/Test_FloatingPointExceptions.cpp

```diff
@@ -13,8 +13,8 @@ SPECTRE_TEST_CASE("Unit.ErrorHandling.FloatingPointExceptions.Invalid",
                   "[ErrorHandling][Unit]") {
   ERROR_TEST();
   enable_floating_point_exceptions();
-  double x = -1.0;
-  double invalid = sqrt(x);
+  volatile double x = -1.0;
```

Comment:
Is this change related to the mac travis build? If not it should be different commit or at least described in this commit message.

Reply:
It is because they were failing until this change

Reply:
In release mode